### ISSUE TITLE
time-consuming？

### ISSUE THREAD

**Issue by wayne807563263** [llmpass/medianDenoise#5]
The denoising process takes 17 seconds on my computer.Is that so time-consuming?

**Comment by llmpass** [comment]
how large is your input image and are you running it on CPU?

**Comment by wayne807563263** [comment]
> how large is your input image and are you running it on CPU?

sorry ,the test is on gpu server Tesla K40m,12G gpu.Does it run fast on your computer?

**Comment by llmpass** [comment]
yes, way faster on my 1080Ti. Are you sure you are using GPU to compute? How is the GPU memory usage? How large is your input image, width and height?

**Comment by wayne807563263** [comment]
> yes, way faster on my 1080Ti. Are you sure you are using GPU to compute? How is the GPU memory usage? How large is your input image, width and height?
 The input image size is 512 by 512, the server gpu memory is  12GB.Maybe my environment is not configured properly.I mainly use PyTorch framework, but I don't know much about Keras. Generally, other neural network denoising time is about 0.02 seconds.

**Comment by llmpass** [comment]
Let's try a small experiment, what about the running time of 1 single median layer on 512x512 input image?
